releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,16 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.0.0 - xx xx, 2023
+
+### Breaking Changes
+* Schema (more information in [Azure App Configuration Kubernetes Provider reference](https://learn.microsoft.com/en-us/azure/azure-app-configuration/reference-kubernetes-provider?tabs=default)):
+    * Rename `keyValues` to `configuration`.
+    * Rename `keyVaults` to `secret` and put it in the root which is at the same level as `configuration`.
+    * Rename `secret.auth.vaults` to `secret.auth.keyVaults`.
+* Upgrade API version from `v1beta1` to `v1`.

```

Comment:
Can we move this up before the breaking change?

Released the stable API version `v1`.